easybuild/easyconfigs/g/googletest/googletest-1.8.1-CrayGNU-18.08.eb

```diff
@@ -0,0 +1,33 @@
+# jg (cscs)
+easyblock = 'CMakeMake'
+
+name = 'googletest'
+version = '1.8.1'
+
+homepage = 'https://github.com/google/googletest'
+description = "Google's C++ test framework"
+
+toolchain = {'name': 'CrayGNU', 'version': '18.08'}
+toolchainopts = {'verbose': False}
+source_urls = ['https://github.com/google/googletest/archive/']
+sources = ['release-%(version)s.tar.gz']
+checksums = ['2e6fbeb6a91310a16efe181886c59596']
+
+builddependencies = [
+    ('CMake', '3.12.0', '', True),
+]
+
+separate_build_dir = True
+
+configopts  = '-DCMAKE_CXX_COMPILER=`which g++` -DCMAKE_C_COMPILER=`which gcc` '
```

Comment:
What about using the cray compiler wrappers?

Reply:
in this special case, i also want to be able to run on the login nodes too but if i compile with the cray wrappers, i won't be able to.

Reply:
Ok, then! 👍